testing/conformance/tests/runner.rs

```diff
@@ -74,76 +75,64 @@ async fn conformance_test_runner() -> anyhow::Result<()> {
         .ok()
         .map(|path| TestTraceExporter::new(Path::new(path.as_str()).to_path_buf()));
 
-    let vector_results = if path.is_file() {
-        let stats = stats.clone();
-        let tracer = tracer.clone();
-        either::Either::Left(
-            iter::once(async move {
-                let res = run_vector(path.clone(), stats, tracer)
-                    .await
-                    .with_context(|| format!("failed to run vector: {}", path.display()))?;
-                anyhow::Ok((path, res))
-            })
-            .map(futures::future::Either::Left),
-        )
+    // Collect test vector files
+    let vector_paths = if path.is_file() {
+        vec![path]
```

Comment:
Double checking - this will not go into some weird behaviour if `path` is a symlink, correct? I mean, `path.is_file` should follow the link correctly, but I'm not sure about the further usage.

Reply:
shouldn't do, the usage is fairly straightforward and the symlink should be transparent from this point onward (we use `File::open(&path)` which should also be fine with symlinks)